docker-compose-proxy.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.3'
```

Comment:
put another way:

The process I typically go through is making local dev as easy as possible because it's hand-typed commands by humans.

Ideally the dev should be able to install Docker, then:
```
git clone xxxxx
docker-compose up
```
Then get to work ;)

CI/CD will be automated so it can have more complex commands that combine compose files with `docker-compose config` and dealing with secrets, env vars, etc.